.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "zigzag_repo/zigzag"]
```

Comment:
Should this be here?

Reply:
I don't think so. I think @RobinGeens used this for testing?

Reply:
This should not be here. Fixed :)